assets/app/views/_overview-deployment.html

```diff
@@ -42,11 +42,11 @@ <h3 class="h4">
       </h3>
     </div>
 
-    <div class="component meta-data">
+    <div class="component meta-data" ng-init="causes = (rc | deploymentCauses)">
```

Comment:
an alternative would be to have the filter set the parsed version as a field on the object... would need to make sure that actually gets reused between invocations, and that it gets updated if the object changes